explorer/scan/local_scanner.go

```diff
@@ -175,7 +181,7 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 			return nil, false, err
 		}
 		log.Debug().Msgf("adding %d asset(s)", len(runtime.Provider.Connection.Inventory.Spec.Assets))
-		assets = append(assets, runtime.Provider.Connection.Inventory.Spec.Assets...)
+		assetCandidates = append(assetCandidates, runtime.Provider.Connection.Inventory.Spec.Assets...)
```

Comment:
i think we also need to guard here, https://github.com/mondoohq/cnquery/compare/vj/gitlab?expand=1